Description: Disable linker option --icf=all as not supported via all linkers
 .
 chromium-browser (84.0.4147.30-0ubuntu1~ppa1~20.04.1) focal; urgency=low
 .
Author: Brad Harper <bjharper@gmail.com>

---
--- chromium-browser-84.0.4147.30.orig/build/config/compiler/BUILD.gn
+++ chromium-browser-84.0.4147.30/build/config/compiler/BUILD.gn
@@ -428,9 +428,9 @@ config("compiler") {
     #}
   }
 
-  if (use_icf) {
-    ldflags += [ "-Wl,--icf=all" ]
-  }
+ #  if (use_icf) {
+ #    ldflags += [ "-Wl,--icf=all" ]
+ #  }
 
   if (is_linux) {
     cflags += [ "-pthread" ]
